### PR TITLE
migrate rsync code to scp

### PR DESCRIPTION
**Pull request by Lassulus**

### CODE REVIEW

File: src/nixos-anywhere.sh [hunk 1]
```diff
@@ -466,11 +466,8 @@ if [[ -n ${extra_files-} ]]; then
     extra_files="$extra_files/"
   fi
   step Copying extra files
-  rsync -rlpv -FF \
```
Comment (Mic92) [inline src/nixos-anywhere.sh:469]: Things to check is if it actually restores permissions/owner correctly.

Comment (Mic92) [reply src/nixos-anywhere.sh:469]: Looks like tests are not happy yet.

Comment (0x4A6F) [reply src/nixos-anywhere.sh:469]: There is also a catch, that the ssh-opt and scp-opt differ (i.e. port).

**Comment by Mic92** [comment]
Closing in favor of https://github.com/nix-community/nixos-anywhere/pull/316

**Pull request closed without merging**